scripts/build.ps1

```diff
@@ -342,58 +344,53 @@ function Publish-Package {
-    Publish-PackageWithRuntimeInternal $dataCollectorProject $TPB_TargetFramework472 $TPB_X64_Runtime false $fullCLRPackage451Dir
+    Publish-PackageWithRuntimeInternal $dataCollectorProject $TPB_TargetFramework472 $TPB_ARM64_Runtime false $fullCLRPackage462Dir
+    Publish-PackageWithRuntimeInternal $dataCollectorProject $TPB_TargetFramework472 $TPB_X64_Runtime false $fullCLRPackage462Dir
     Publish-PackageInternal $dataCollectorProject $TPB_TargetFrameworkCore20 $coreCLR20PackageDir
 
     ################################################################################
     # Publish testhost
     Write-Log "Package: Publish testhost\testhost.csproj"
-    Publish-PackageInternal $testHostProject $TPB_TargetFramework451 $testhostFullPackageDir
+    Publish-PackageInternal $testHostProject $TPB_TargetFramework462 $testhostFullPackageDir
     Publish-PackageInternal $testHostProject $TPB_TargetFrameworkCore20 $testhostCore20PackageDir
     Publish-PackageInternal $testHostProject $TPB_TargetFrameworkCore10 $testhostCore10PackageDir
     Publish-PackageInternal $testHostProject $TPB_TargetFrameworkCore20 $testhostUapPackageDir
     Publish-PackageWithRuntimeInternal $testHostProject $TPB_TargetFrameworkCore20 $TPB_X64_Runtime false $testhostCore20PackageTempX64Dir
     Publish-PackageWithRuntimeInternal $testHostProject $TPB_TargetFrameworkCore10 $TPB_X64_Runtime true $testhostCore10PackageTempX64Dir
 
     Write-Log "Package: Publish testhost.x86\testhost.x86.csproj"
-    Publish-PackageInternal $testHostx86Project $TPB_TargetFramework451 $testhostFullPackageDir
+    Publish-PackageInternal $testHostx86Project $TPB_TargetFramework462 $testhostFullPackageDir
     Publish-PackageWithRuntimeInternal $testHostx86Project $TPB_TargetFrameworkCore20 $TPB_X86_Runtime false $testhostCore20PackageTempX86Dir
     Publish-PackageWithRuntimeInternal $testHostx86Project $TPB_TargetFrameworkCore10 $TPB_X86_Runtime true $testhostCore10PackageTempX86Dir
 
     Write-Log "Package: Publish testhost.arm64\testhost.arm64.csproj"
-    Publish-PackageInternal $testHostarm64Project $TPB_TargetFramework451 $testhostFullPackageDir
+    Publish-PackageInternal $testHostarm64Project $TPB_TargetFramework462 $testhostFullPackageDir
     Publish-PackageWithRuntimeInternal $testHostarm64Project $TPB_TargetFrameworkCore20 $TPB_ARM64_Runtime false $testhostCore20PackageTempARM64Dir
     Publish-PackageWithRuntimeInternal $testHostarm64Project $TPB_TargetFrameworkCore10 $TPB_ARM64_Runtime true $testhostCore10PackageTempARM64Dir
 
-    # Copy the .NET multitarget testhost exes to destination folder (except for net451 which is the default)
-    foreach ($tfm in "net452;net46;net461;net462;net47;net471;net472;net48" -split ";") {
+    # Copy the .NET multitarget testhost exes to destination folder (except for net462 which is the default)
+    foreach ($tfm in "net47;net471;net472;net48" -split ";") {
```

Comment:
Merge the 3 loops into one and add specific group comment.

Reply:
Why?